.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+## Pull requests
```

Comment:
We added on Starknet.js something like a warning for low quality PR-s that we will not consider them in detail nor merge
https://github.com/starknet-io/starknet.js/pull/1117

Reply:
Added a modified version in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/454/commits/179056059ec9c9ddd4be27554813d6d608a27350